engine/installation/linux/docker-ce/centos.md

```diff
@@ -0,0 +1,258 @@
+
+1.  Update the `yum` package index.
+
+    ```bash
+    $ sudo yum makecache fast
+    ```
+
+    If this is the first time you have refreshed the package index since adding
+    the Docker repositories, you will be prompted to accept the GPG key, and
+    the key's fingerprint will be shown. Verify that the fingerprint is
+    correct, and if so, accept the key. The fingerprint should match
+    `060A 61C5 1B55 8A7F 742B  77AA C52F EB6B 621E 9F35`.
+
+2.  Install the latest version of Docker CE, or go to the next step to install a
+    specific version.
+
+    ```bash
+    $ sudo yum install docker-ce
+    ```
+
+    > **Warning**: If you have multiple Docker repositories enabled, installing
+    > or updating without specifying a version in the `yum install` or
+    > `yum update` command will always install the highest possible version,
+    > which may not be appropriate for your stability needs.
+
+3.  On production systems, you should install a specific version of Docker CE
+    instead of always using the latest. List the available versions. This
+    example uses the `sort -r` command to sort the results by version number,
+    highest to lowest, and is truncated.
+
+    > **Note**: This `yum list` command only shows binary packages. To show
+    > source packages as well, omit the `.x86_64` from the package name.
+
+    ```bash
+    $ yum list docker-ce.x86_64  --showduplicates |sort -r
+
+    docker-ce.x86_64  {{ minor-version }}.0.el7                               docker-ce-stable  
+    ```
+
+    The contents of the list depend upon which repositories are enabled, and
+    will be specific to your version of CentOS (indicated by the `.el7` suffix
+    on the version, in this example). Choose a specific version to install. The
+    second column is the version string. The third column is the repository
+    name, which indicates which repository the package is from and by extension
+    its stability level. To install a specific version, append the version
+    string to the package name and separate them by a hyphen (`-`):
+
+    ```bash
+    $ sudo yum install docker-ce-<VERSION>
+    ```
+
+4.  Start Docker.
+
+    ```bash
+    $ sudo systemctl start docker
+    ```
+
+5.  Verify that `docker` is installed correctly by running the `hello-world`
+    image.
+
+    ```bash
+    $ sudo docker run hello-world
+    ```
+
+    This command downloads a test image and runs it in a container. When the
+    container runs, it prints an informational message and exits.
+
+Docker CE is installed and running. You need to use `sudo` to run Docker
+commands. Continue to [Linux postinstall](linux-postinstall.md) to allow
+non-privileged users to run Docker commands and for other optional configuration
+steps.
+
+#### Upgrade Docker CE
+
+To upgrade Docker CE, first run `sudo yum makecache fast`, then follow the
+[installation instructions](#install-docker), choosing the new version you want
+to install.
+
+### Install from a package
```

Comment:
Oh, just saw your comment here; not sure I understand your question though :blush: